qa/TL1_videoreader_benchmark/test.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash -e
+
+one_config_only=true
+target_dir=./qa/TL1_videoreader_benchmark/
+
+do_once() {
+  apt-get update
+  apt-get install -y ffmpeg linux-tools-$(uname -r)
+
+  cp -r "${DALI_EXTRA_PATH}"/db/video_resolution .
+
+  BENCHMARK_BIN=dali_benchmark.bin
+  FULL_PATH=`find "../../" -name $BENCHMARK_BIN`
```

Comment:
Benchmark bin is shipped as a part of DALI whl.
Please check https://github.com/NVIDIA/DALI/blob/master/qa/TL0_rn50-benchmarks/test.sh to see how to find the binary.